automation/symbols-generation/upload_symbols.py

```diff
@@ -5,29 +5,32 @@
 
```

Comment:
I assume symbols uploading is still OK?

Reply:
Yes. I just moved the way we consume credentials. Tested locally and in staging releases. Unfortunately, I can't test production push until we actually merge but all the local graphs suggest it's working smooth.